src/components/accessories/permissionWrapper/PermissionWrapper.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FC } from "react";
+import { useAlternativePermissions } from "../../../libraries/permissionUtils/useAlternativePermissions";
+import { usePermission } from "../../../libraries/permissionUtils/usePermission";
+import { AlertBox } from "../alertBox/AlertBox";
+import { CustomPermissionDenied } from "../customPermissionDenied/CustomPermissionDenied";
+import { PermissionDenied } from "../permissionDenied/PermissionDenied";
+import { IOwnProps } from "./types";
+
+export const PermissionWrapper: FC<IOwnProps> = ({
+  fallback = <CustomPermissionDenied />,
+  permissions,
+  children,
+}) => {
+  const canAccess = useAlternativePermissions(permissions);
+  return <>{canAccess === true ? children : fallback}</>;
+};
```

Comment:
We already implemented a wrapper component called "Permission.tsx" inside libraries/permissionUtils.

Why did you need to create another one?